autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/ApiActivityStatus.java

```diff
@@ -0,0 +1,17 @@
+package com.sequenceiq.periscope.api.model;
+
+public enum ApiActivityStatus {
```

Comment:
Is this the intended approach? Introduces complete redundancy with `ActivityStatus`